tests/python/unittest/test_operator.py

```diff
@@ -8655,6 +8656,17 @@ def test_add_n():
     assert_almost_equal(rslt.asnumpy(), add_n_rslt.asnumpy(), atol=1e-5)
 
 
+def test_get_all_registered_operators():
+    ops = get_all_registered_operators()
+    ok_(isinstance(ops, list))
+    ok_(len(ops) > 0)
```

Comment:
Could we check for one operator of which we certainly know it will exist?

Reply:
ok, I thought in the future we might dynamically build with only the operators needed, so I didn't want to make this asumption. For now we can assume that some basic op will be present.